### PR TITLE
Buffs Pepperspray with more eye blurring/blindness

### PR DESCRIPTION
**Pull request by Poojawa**
## About The Pull Request

because honestly it's useless.

## Why It's Good For The Game

being blinded by pepperspray should have a lasting effect instead of being a chemical based flash.

### CODE REVIEW

**Comment by Trilbyspaceclone** [comment]
Ah yes, waring sunglasses. 
Makes pepper spray *and* flashes useless
I rather make it just be "If Hide identity" check...

**Review by silicons — CHANGES_REQUESTED** [review]
this is ridiculously, ridiculously high.

**Comment by Poojawa** [comment]
have you ever actually been sprayed with pepper spray before? it's not something you blink a few times and can see perfectly again.

**Comment by silicons** [comment]
@Poojawa No, but this is a game, and if your argument becomes realism it's an invalid argument.

**Comment by silicons** [comment]
If pepperspray is so hard to land that it justifies *this* it's bad, but this is just, no. 
65 ticks of blur and 10 ticks of blind is a hard no, this makes it from "hard to land weapon that's useless" to "hard to land weapon that's almost an instant win if they're not protected" 
it screams bad design sorry, if we need to make projectile-based reagent sprays/shit so be it, hell i can probably make that if needed, but this kinda stuff sucks and i already made one mistake when beam rifles were made.

**Comment by silicons** [comment]
tl;dr maybe pepperspray should be far easier to use and not so glitchy + gimmicky (iirc sprays are weird for hit calculations/etc) than it being a clumsy hit-like-a-truck weapon.
